commands/credentials/destroy.js

```diff
@@ -11,8 +11,15 @@ function * run (context, heroku) {
   const {app, args, flags} = context
   let cred = flags.name
 
+  if (cred === 'default') {
+    throw new Error('Default credential cannot be destroyed.')
+  }
+
   let db = yield fetcher.addon(app, args.database)
-  if (util.starterPlan(db)) throw new Error('This operation is not supported by Hobby tier databases.')
+  if (util.starterPlan(db)) {
+    throw new Error(`Could not find credential ${cred} for database ${db.name}.
+Only one default credential is supported for Hobby tier databases.`)
```

Comment:
The first part of this feels superfluous. Maybe just `Only the default credential is supported for Hobby tier databases.`?